[StructuralMechanicsApplication] CRBeam test failing

~~~sh
======================================================================
FAIL: test_cr_beam_dynamic_consistent_mass_matrix (test_patch_test_cr_beam.TestCrBeam2D2N)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vicente/src/Kratos/applications/StructuralMechanicsApplication/tests/test_patch_test_cr_beam.py", line 952, in test_cr_beam_dynamic_consistent_mass_matrix
    self._check_results_dynamic(mp,time_i,nr_nodes,time_step)
  File "/home/vicente/src/Kratos/applications/StructuralMechanicsApplication/tests/test_patch_test_cr_beam.py", line 813, in _check_results_dynamic
    self.assertAlmostEqual(disp_y_analytical[time_step], disp_y_simulated)
AssertionError: -4.769300755191701e-05 != -4.162518355093401e-05 within 7 places (6.067824000983001e-06 difference)

~~~

--- a/applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_2D2N.cpp
+++ b/applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_2D2N.cpp
@@ -153,7 +153,7 @@ void CrBeamElement2D2N::CalculateMassMatrix(MatrixType &rMassMatrix,
   const double A = this->GetProperties()[CROSS_AREA];
   const double rho = StructuralMechanicsElementUtilities::GetDensityForMassMatrixComputation(*this);
 
-  bool use_consistent_mass_matrix = false;
+  bool use_consistent_mass_matrix = true;
   if (this->GetProperties().Has(USE_CONSISTENT_MASS_MATRIX)) {
     use_consistent_mass_matrix = GetProperties()[USE_CONSISTENT_MASS_MATRIX];
   }
@@ -197,7 +197,7 @@ void CrBeamElement2D2N::CalculateMassMatrix(MatrixType &rMassMatrix,
   else
   {
     const double lumped_mass = A * L * rho;
-    
+
      // w.r.t. Felippa - Chapter 31: LUMPED AND CONSISTENT MASS MATRICES - p.31–10
     double alpha = 0.00;
     if (this->GetProperties().Has(LUMPED_MASS_ROTATION_COEFFICIENT)) {

